image_explorer/image_explorer.py

```diff
@@ -373,6 +373,10 @@ def _get_hotspots(self, xmltree, absolute_urls=False):
             if not hotspot.y.endswith('%'):
                 hotspot.y += 'px'  # px is deprecated as it is not responsive
 
+            hotspot.visited = False
+            if hotspot.item_id in self.opened_hotspots:
+                hotspot.visited = True
+
```

Comment:
nit: This can be simplified to:

```suggestion
hotspot.visited = hotspot.item_id in self.opened_hotspots
```

If you prefer your original style though, that's perfectly fair :-)